MediaBrowser.MediaEncoding/Subtitles/JsonWriter.cs

```diff
@@ -33,7 +34,10 @@ public void Write(SubtitleTrackInfo info, Stream stream, CancellationToken cance
                     writer.WriteEndObject();
                 }
 
+                writer.WriteEndArray();
                 writer.WriteEndObject();
+
+                writer.Flush();
```

Comment:
Why call flush instead of close? I'm not sure we would use this stream more than once for any given request for subtitles.

Reply:
There is no close method
(and technically this flush isn’t needed, it gets flushed on dispose)